src/main/kotlin/com/example/noswimmingserver/domain/reading_journal/domain/repository/ReadingJournalRepository.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.noswimmingserver.domain.reading_journal.domain.repository
```

Comment:
🚫 **[ktlint]** <package-name> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name must not contain underscore (cannot be auto-corrected)